spec/features/maths_and_physics_claim_spec.rb

```diff
@@ -165,6 +170,55 @@
     expect(page).to have_text(I18n.t("questions.qts_award_year"))
   end
 
+  scenario "Supply teacher claims for Maths and Physics, employed directly by school with a contract to teach for an entire term" do
```

Comment:
I'm wondering with these 'alernative journey' specs if we do something along the lines of what we do for the Changing Answers specs (https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/spec/features/changing_answers_spec.rb#L8-L11) in that we start the claim (so we have a claim in memory), then update the claim's details up to the part we're testing, and then visit that path directly. This would solve a lot of duplication, and speed up the tests too. 

Also, rather than testing up until the end of the journey we can just test whether the claim is still eligible with something like:

```ruby
expect(claim.reload.ineligbile?).to eq(false)
```

Reply:
I think it's a good idea. I think that perhaps we could leave it as it is here, and revisit these specs once we see how much of these additional journeys we explore in the (yet to come) changing answers specs for Maths and Physics, what do you think?

Reply:
Yeah, makes sense 👍